llvm/include/llvm/BinaryFormat/SFrame.h

```diff
@@ -104,14 +104,8 @@ template <endianness E> struct Header {
   detail::packed<uint32_t, E> FREOff;
 };
 
-template <endianness E> struct FuncDescEntry {
-  detail::packed<int32_t, E> StartAddress;
-  detail::packed<uint32_t, E> Size;
-  detail::packed<uint32_t, E> StartFREOff;
-  detail::packed<uint32_t, E> NumFREs;
-  detail::packed<uint8_t, E> Info;
-  detail::packed<uint8_t, E> RepSize;
-  detail::packed<uint16_t, E> Padding2;
+struct FDEInfo {
+  uint8_t Info;
```

Comment:
FREInfo uses the endian types, even though they're kind of unnecessary. The only reason for that was consistency, so I don't really have a problem with using uint8_t directly, but if you do that here, please also change FREInfo to match.